[2024-12-01T00:00:09.551+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-01T00:00:09.707+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_variables_financieras.consumo_stg_var1_reservas scheduled__2024-11-30T00:00:00+00:00 [queued]>
[2024-12-01T00:00:10.191+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_variables_financieras.consumo_stg_var1_reservas scheduled__2024-11-30T00:00:00+00:00 [queued]>
[2024-12-01T00:00:10.195+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-01T00:00:10.269+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): consumo_stg_var1_reservas> on 2024-11-30 00:00:00+00:00
[2024-12-01T00:00:10.320+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2331) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-01T00:00:10.321+0000] {standard_task_runner.py:63} INFO - Started process 2337 to run task
[2024-12-01T00:00:10.318+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'process_variables_financieras', 'consumo_stg_var1_reservas', 'scheduled__2024-11-30T00:00:00+00:00', '--job-id', '382', '--raw', '--subdir', 'DAGS_FOLDER/etl_prueba.py', '--cfg-path', '/tmp/tmpegdm82x_']
[2024-12-01T00:00:10.333+0000] {standard_task_runner.py:91} INFO - Job 382: Subtask consumo_stg_var1_reservas
[2024-12-01T00:00:10.501+0000] {task_command.py:426} INFO - Running <TaskInstance: process_variables_financieras.consumo_stg_var1_reservas scheduled__2024-11-30T00:00:00+00:00 [running]> on host 5e909a8df6c4
[2024-12-01T00:00:10.902+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='process_variables_financieras' AIRFLOW_CTX_TASK_ID='consumo_stg_var1_reservas' AIRFLOW_CTX_EXECUTION_DATE='2024-11-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-30T00:00:00+00:00'
[2024-12-01T00:00:10.904+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-01T00:00:10.940+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2024-12-01T00:00:11.012+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-12-01T00:00:11.019+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-01T00:00:11.040+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=process_variables_financieras, task_id=consumo_stg_var1_reservas, run_id=scheduled__2024-11-30T00:00:00+00:00, execution_date=20241130T000000, start_date=20241201T000009, end_date=20241201T000011
[2024-12-01T00:00:11.137+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-01T00:00:11.209+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-01T00:00:11.223+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
